### PR TITLE
The ultimate memechem part 1 (to be continued ?????????????)

### PR DESCRIPTION
**Pull request by KayeArray**
Adds meme drug flipout which flips everything including inanimate objects.

Is pretty annoying to overdose on and slightly dangerous if addicted.

Also for some reason overdose doesn't do a constant flip and keeps alternating between the normal flip and the super long one, anyone have ideas on how to fix?

### CODE REVIEW

**Comment by Spacedong** [comment]
cool

**Comment by KayeArray** [comment]
i wil make video

File: code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm [hunk 1]
```diff
@@ -565,4 +565,90 @@
...
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(prob(70))
+			H.SpinAnimation(16,100)
+			H.Dizzy(20)
+			if(M.canmove && !istype(M.loc, /atom/movable))
+				for(var/i = 0, i < 4, i++)
+				step(M, pick(cardinal))
+		if(prob(15))
+			M << "<span class='danger'>The flipping is so intense you begin to tire!.</span>"
```
Comment (ghost) [inline code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm]: "!."
Remove please

**Comment by ghost** [comment]
Why not add this effect to rotatium? Nobody actually uses rotatium, so this would be a nice update.

**Comment by KayeArray** [comment]
Because rotatium is already trippy.

**Comment by ArcLumin** [comment]
>no transform =* -1

kill me

**Comment by KayeArray** [comment]
?

**Comment by ArcLumin** [comment]
Also we have another PR with this exact same chem in in bro; https://github.com/HippieStationCode/HippieStation13/pull/3286

**Comment by KayeArray** [comment]
he removed it silly mcnilly

**Comment by ArcLumin** [comment]
Why the fuck would he remove the best chem in that bunch

**Comment by KayeArray** [comment]
because this is so much better, his was an occasional flip while this is flip spam and then flip loops if you get overdosed and literally insane flip dynamo if you get addicted and it makes all obj's flip as well

**Comment by ArcLumin** [comment]
Yeah, okay. Make it do a transform =* -1 on overdose as well or some shit and it'll be greato in my book

**Comment by ArcLumin** [comment]
Sorry, transform *= -1

**Comment by ArcLumin** [comment]
Code's good

**Comment by KayeArray** [comment]
wew got the merge conflicts fixed.

**Comment by ghost** [comment]
nice cool i love memes
+1

**Comment by SpankMasterson** [comment]
+1

**Pull request merged**